pytential/symbolic/primitives.py

```diff
@@ -539,6 +548,22 @@ def __getinitargs__(self):
         return (self.dofdesc,)
 
 
+class _ShapeDiscretizationProperty(DiscretizationProperty):
```

Comment:
Not sure I'm loving this. We already have a conditional. Why not use that? All we'd need in addition is a "Is the reference element in this group a ..." predicate.

Reply:
That's a lot nicer! Added something like that in 51ec319.